### PR TITLE
chore(price-feeder): update example config

### PR DESCRIPTION
**Pull request by adamewozniak**
## Description

Updates the price-feeder's example config with options in preparation for mainnet.
Locally I don't see any errors, please double check me 😄 

closes: #1270

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] added `!` to the type prefix if API or client breaking change
- [ ] added appropriate labels to the PR
- [ ] targeted the correct branch (see [PR Targeting](https://github.com/umee-network/umee/blob/main/CONTRIBUTING.md#pr-targeting))
- [ ] provided a link to the relevant issue or specification
- [ ] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items._

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed state machine logic
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

File: price-feeder/price-feeder.example.toml [hunk 1]
```diff
@@ -8,32 +8,55 @@ write_timeout = "20s"
...
+  "ftx"
 ]
 quote = "USD"
 
 [[currency_pairs]]
 base = "USDT"
 providers = [
   "kraken",
-  "gate",
+  "ftx",
```
Comment (RafilxTenfen) [inline price-feeder/price-feeder.example.toml:40]: :tada:

File: price-feeder/price-feeder.example.toml [hunk 2]
```diff
@@ -8,32 +8,55 @@ write_timeout = "20s"
...
+
+[[currency_pairs]]
+base = "UMEE"
 providers = [
-  "binance",
+  "okx",
+  "gate"
 ]
-quote = "USDT"
+  quote = "USDT"
```
Comment (adamewozniak) [inline price-feeder/price-feeder.example.toml:27]:
```suggestion
quote = "USDT"
```

Comment (RafilxTenfen) [inline price-feeder/price-feeder.example.toml]:
```suggestion
quote = "USDT"
```

**Review by RafilxTenfen — APPROVED** [review]
worked for me ^^

**Comment by RafilxTenfen** [comment]
@adamewozniak forgot to mention, should we add `provider_timeout = "5s"` ?

**Comment by adamewozniak** [comment]
> @adamewozniak forgot to mention, should we add `provider_timeout = "5s"` ? 
> 
> 

Imo this depends on where the validator's server is

**Pull request merged**